Source/JavaScriptCore/bytecode/AccessCaseSnippetParams.h

```diff
@@ -45,7 +45,7 @@ class AccessCaseSnippetParams final : public SnippetParams {
     }
 
     class SlowPathCallGenerator {
-        WTF_DEPRECATED_MAKE_FAST_ALLOCATED(SlowPathCallGenerator);
+        WTF_MAKE_TZONE_ALLOCATED(SlowPathCallGenerator);
```

Comment:
This is an abstract class.  Let's make this `WTF_FORBID_HEAP_ALLOCATION` instead and save on the overhead of pretending that this is TZone allocated.